media-video/obs-studio/obs-studio-9999.ebuild

```diff
@@ -3,16 +3,17 @@
 
 EAPI=8
 
-CMAKE_REMOVE_MODULES_LIST=( FindFreetype )
-LUA_COMPAT=( luajit )
-PYTHON_COMPAT=( python3_{9..12} )
+CMAKE_REMOVE_MODULES_LIST=(FindFreetype)
+LUA_COMPAT=(luajit)
+PYTHON_COMPAT=(python3_{9..12})
```

Comment:
Editor automatically making style changes perhaps?

Reply:
Yes was the editor changing the style